datasafe-test-storages/src/main/java/de/adorsys/datasafe/teststorage/WithStorageProvider.java

```diff
@@ -255,6 +270,57 @@ protected static StorageDescriptor s3() {
         );
     }
 
+    protected static Map<String, StorageDescriptor> getMultiS3Bucket(List<String> users){
+        if(amazons3BucketCount == null){
```

Comment:
Please format code according to Oracle Java formatting standard 
https://www.oracle.com/technetwork/java/codeconvtoc-136057.html

Reply:
done